packages/@ourworldindata/grapher/src/lineCharts/LineChart.tsx

```diff
@@ -1160,11 +1160,14 @@ export class LineChart
 
     @computed private get dualAxis(): DualAxis {
         return new DualAxis({
-            bounds: this.boundsWithoutColorLegend.padRight(
-                this.lineLegendDimensions
-                    ? this.lineLegendDimensions.width
-                    : this.defaultRightPadding
-            ),
+            bounds: this.boundsWithoutColorLegend
+                .padRight(
+                    this.lineLegendDimensions
+                        ? this.lineLegendDimensions.width
+                        : this.defaultRightPadding
+                )
+                // top padding leaves room for tick labels
+                .padTop(6),
```

Comment:
This is one of the instances where I included some padding on top to prevent charts to overflow. Not really sure this is the best way of doing this...